pkg/cluster/api/pdapi.go

```diff
@@ -274,7 +275,8 @@ func (pc *PDClient) GetDashboardAddress() (string, error) {
 		return "", errors.AddStack(err)
 	}
 
-	return pdConfig.PDServerCfg.DashboardAddress, nil
+	r := strings.NewReplacer("http://", "", "https://", "")
+	return r.Replace(pdConfig.PDServerCfg.DashboardAddress), nil
```

Comment:
This affects PR #393 , not sure whether this is acceptable, if doesn't, we need to move this part logic out.

Reply:
Yes, it affects #393 as well as our documentations. We'd better leave it unchanged, since it will be used to construct a full Dashboard access URL later.

Reply:
Done.